src/chat.test.ts

```diff
@@ -21,23 +21,51 @@ test('non streaming chat', async () => {
 });
 
 test('streaming chat', async () => {
-  const chat = await replitai.chatStream({
+  const result = await replitai.chatStream({
     model: 'chat-bison',
     messages: [{ content: 'what is the meaning of life', author: 'user' }],
     temperature: 0.5,
-    maxOutputTokens: 1024,
+    maxOutputTokens: 128,
   });
 
-  expect(chat.error).toBeFalsy();
+  expect(result.error).toBeFalsy();
 
-  if (!chat.ok) {
+  if (!result.ok) {
     throw new Error('wat');
   }
 
-  for await (const { message } of chat.value) {
+  for await (const { message } of result.value) {
     expect(message).toMatchObject({
       content: expect.any(String),
       author: expect.any(String),
     });
   }
 });
+
+test('chat with multiple choices', async () => {
+  const result = await replitai.chatMultipleChoices({
+    model: 'chat-bison',
+    messages: [{ content: 'what is the meaning of life', author: 'user' }],
+    temperature: 0.5,
+    maxOutputTokens: 128,
+    choicesCount: 4,
+  });
+
+  expect(result.error).toBeFalsy();
+
+  if (!result.ok) {
+    throw new Error('wat');
+  }
```

Comment:
optional:

```suggestion

  expect(result.ok).toBeTrue();
```

feels weird to throw an error in the middle of a sea of `expect`s.

this s

Reply:
yeah, typescript cannot infer from `expect` :(

Reply:
otherwise the error check would've sufficed